### PR TITLE
Adds RUNNER_GROUP argument to the runner registration

### LINKED ISSUE #137: Support for Runner Groups
Great operator, thanks. Perhaps I've just not figured out how to configure it, but is there a way to assign the `RunnerDeployment` to a Runner Group for an Org? I've not seen use of the API in a repo code-search;  `/orgs/{org}/actions/runner-groups`

https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#self-hosted-runner-groups

### PR DESCRIPTION
**Pull request by damacus**
Adds the ability to register a runner to a predefined runner_group

Resolves #137

### CODE REVIEW

**Comment by damacus** [comment]
Fairly sure I've missed something here, but this is a first crack at adding RUNNER_GROUP into the controller 👍 

I've started the integration tests, but I'd appreciate a bit of direction on how to test this grouping works.

**Review by jbuettnerbild — APPROVED** [review]

**Comment by jbuettnerbild** [comment]
its very simple:
doku here: https://docs.github.com/en/free-pro-team@latest/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups
maybe you can also update the Readme

**Comment by damacus** [comment]
I was more thinking about automated testing, I've actually got runner groups working and registering correctly using https://github.com/myoung34/docker-github-actions-runner (however that doesn't yet support GitHub Apps authentication).

Will update the README shortly 👍

File: README.md [hunk 1]
```diff
@@ -372,22 +373,41 @@ jobs:
 
 Note that if you specify `self-hosted` in your workflow, then this will run your job on _any_ self-hosted runner, regardless of the labels that they have.
 
+## Runner Groups
+
+Runner groups can be used to limit which repositories are able to use the GitHub Runne at an Organisation level.
```
Comment (jbuettnerbild) [inline README.md:378]: typo. -Runne-

Comment (damacus) [reply README.md]: That was a long Friday. Thanks for spotting that

**Review by mumoshu — APPROVED** [review]
Awesome work! Thanks for your effort @damacus and your review @jbuettnerbild ☺️

**Pull request merged**